blocks/form/gdpr-confirmation.js

```diff
@@ -0,0 +1,78 @@
+// get url parameters
+const curUrl = window.location.href;
+const urlParams = new URLSearchParams(window.location.search);
+const paramdata = urlParams.get('data');
+const paraminit = urlParams.get('init');
+
+function hasQueryParams() {
+  return curUrl.toString().includes('?');
```

Comment:
more straightforward would be `return !!location.search;` you can remove the `const curUrl = window.location.href;` line above then